src/oci.h

```diff
@@ -137,6 +137,11 @@
 /* Path to the stateless passwd file. */ 
 #define STATELESS_PASSWD_PATH "/usr/share/defaults/etc/passwd"
 
+/* Offset to add to the interface index for assigning the pci slot.
+ * First 3 slots are in use for pc-lite machine type
+ */
+#define PCI_OFFSET 4
```

Comment:
@jodh-intel The pci slots currently in use are:
  PCI: slot 0 function 0 => in use by pci-lite-device
  PCI: slot 1 function 0 => in use by PM_LITE
  PCI: slot 2 function 0 => in use by virtio-9p-pci
  PCI: slot 3 function 0 => in use by virtio-serial-pci

The first two 0 and 1 seem to be used by qemu for the pc-lite machine type, the last two are used by us for the 9p mount and for the virtual console respectively. I'll be updating the comments to reflect this.

I am inclined to set the offset to a higher value (maybe 8) to take into account any available free slots being assigned by qemu or used by us in the future. What do you think?